source/getting-started.txt

```diff
@@ -0,0 +1,72 @@
+.. _relational-migrator-getting-started:
+
+===============
+Getting Started
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+Relational Migrator allows you to bring your relational workloads 
+to MongoDB. 
```

Comment:
(short description)
You might also summarize a few of the main Relational Migrator features and capabilities. I'm happy to chat about this if that's helpful.